components/tag/TagSimple.js

```diff
@@ -1,6 +1,9 @@
-export default function TagSimple({ name }) {
+import { classNames } from "@services/utils/classNames";
+
+export default function TagSimple({ name, isSelected }) {
+ 
   return (
-    <div className="text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1">
+    <div className={classNames("text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1", isSelected && "border-tertiary-medium")}>
```

Comment:
I think the formatting is not correct, please check you have Prettier VScode plugin installed...

```js
import { classNames } from "@services/utils/classNames";

export default function TagSimple({ name, isSelected }) {
  return (
    <div
      className={classNames(
        "text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1",
        isSelected && "border-tertiary-medium"
      )}
    >
      {name}
    </div>
  );
}
```